NGitLab/Models/PackageType.cs

```diff
@@ -0,0 +1,17 @@
+namespace NGitLab.Models
+{
+    public enum PackageType
+    {
+        all,
```

Comment:
Use C# name (e.g. `Default`) and the `EnumMember(Value = "")]` attribute to map the actual name